velox/type/Type.cpp

```diff
@@ -897,6 +899,16 @@ std::string IntervalDayTimeType::valueToString(int64_t value) const {
   return buf;
 }
 
+std::string IntervalYearMonthType::valueToString(int32_t value) const {
+  static const char* kIntervalFormat = "%d %d";
+  int32_t remainMonths = value;
+  const int32_t years = remainMonths / kMonthInYear;
+  remainMonths -= years * kMonthInYear;
+  char buf[32];
+  snprintf(buf, sizeof(buf), kIntervalFormat, years, remainMonths);
```

Comment:
We've generally used std::ostringstream and fmt::format (instead of snprintf) for such code.

An option for you is like:

std::ostringstream oss;
oss << fmt::format("{} {}", years, remainMonths);
return oss.str();

That would also remove variable kIntervalFormat.